### DISCUSSION TITLE
Saving JSON in wrong place

### DISCUSSION THREAD

**Discussion by mauro-baptista** [MohmmedAshraf/laravel-translations#72] [Ideas]
I'm using a lot of keys that are just texts, for instance `__('Device Count')`. This kind of translation only works with json file, but when I've tried to publish it, it saved the information inside the `lang\en` directory, when it should be saved in the `lang` directory. 

I believe we could change `src/TranslationsManager.php` in the `export` method to 

```php
    $langPath = match(true) {
        $download => storage_path("app/translations/$locale/$file"),
        $this->filesystem->extension(lang_path("$locale/$file")) == 'php' => lang_path("$locale/$file"),
        $this->filesystem->extension(lang_path("$file")) == 'json' => lang_path("$file"),
    };
```

instead of `$langPath = $download ? storage_path("app/translations/$locale/$file") : lang_path("$locale/$file");`.

Did you had issues saving it in json?

**Comment by MohmmedAshraf** [comment]
Hi @mauro-baptista , Try the latest release and let me know if it works for you.

**Reply by mauro-baptista** [reply]
It is working now. Thanks.
